app/views/donations/index.html.slim

```diff
@@ -1,10 +1,23 @@
 .card.card-inverse.card-info.text-xs-center
-  .card.card-inverse.card-info.text-xs-center
-    .card-block
-      h1 Total Donations for #{Date.today.year}
-      h1 $#{@donations.donations_per_year(Date.current.year).sum(:amount).round}
+  .card-block
+    h1 Total Donations for #{Date.today.year}
+    h1 $#{@donations.donations_per_year(Date.current.year).sum(:amount).round}
 = area_chart @donations.includes(:cause).where("donations.created_at BETWEEN ? AND ?", Date.current.beginning_of_year, Date.current.end_of_year).group("causes.name").group_by_week("donations.created_at").sum(:amount)
 
+.row.margin-top
+  - @causes.each do |cause|
+    .col-xs-6.col-md-3
+      .card.card-outline-primary.text-xs-center
+        .card-header = cause.name
+        .card-block
+          h4.card-title Top Donor:
+          - top_donation = cause.donations.where("donations.created_at BETWEEN ? AND ?", Date.current.beginning_of_year, Date.current.end_of_year).order("amount DESC").first
```

Comment:
Qualifying the `donations` table shouldn't be required in this query.

Also, we should consider moving this somewhere else. Assigning variables and running queries in the view is usually an indication that we're missing some object. 😀